src/vs/workbench/contrib/terminal/browser/terminalActions.ts

```diff
@@ -1711,6 +1711,26 @@ export function registerTerminalActions() {
 			}
 		}
 	});
+	registerAction2(class extends Action2 {
+		constructor() {
+			super({
+				id: TerminalCommandId.ShowOrFocusHover,
+				title: terminalStrings.showOrFocusHover,
+				f1: false,
+				category,
+				precondition: ContextKeyExpr.or(ContextKeyExpr.or(TerminalContextKeys.processSupported, TerminalContextKeys.terminalHasBeenCreated), TerminalContextKeys.isOpen),
+				keybinding: {
+					primary: KeyChord(KeyMod.CtrlCmd | KeyCode.KeyK, KeyMod.CtrlCmd | KeyCode.KeyI),
```

Comment:
this keybinding does not work for me when I have focused the terminal. It works if I have focused the tabs view. The screen reader users I've worked with usually have only one terminal open. 
<img width="1551" alt="Screenshot 2023-03-16 at 10 19 45 AM" src="https://user-images.githubusercontent.com/29464607/225701857-911835c8-a2b2-49fe-b57d-ea690c2e6e74.png">

Reply:
Was this before my changes I pushed this morning? https://github.com/microsoft/vscode/pull/177283/commits/e563b590eafc833da1c9e7724a551514293775ee

Reply:
this is with that change / just pulled the latest

Reply:
Might have been the missing commands to skip shell, testing on mac...